src/components/Tabs/tests/Tabs.test.tsx

```diff
@@ -1,18 +1,33 @@
 import * as React from 'react';
-import {mountWithAppProvider} from 'test-utilities';
-import {Tab, Panel} from '../components';
+import {mountWithAppProvider, trigger} from 'test-utilities';
+import {noop} from '@shopify/javascript-utilities/other';
+import {ReactWrapper} from 'enzyme';
+import {Tab, Panel, TabMeasurer, List} from '../components';
 import Tabs, {Props} from '../Tabs';
+import {getVisibleAndHiddenTabIndices} from '../utilities';
+import Popover from '../../Popover';
 
 describe('<Tabs />', () => {
-  let tabs: Props['tabs'];
+  const tabs: Props['tabs'] = [
+    {content: 'Tab 1', id: 'tab-1'},
+    {content: 'Tab 2', id: 'tab-2'},
+  ];
 
-  beforeEach(() => {
-    tabs = [{content: 'Tab 1', id: 'tab-1'}, {content: 'Tab 2', id: 'tab-2'}];
+  const mockProps = {
+    selected: 0,
+    tabs,
+    onSelect: noop,
+  };
+
+  afterEach(() => {
+    if (document.activeElement) {
+      (document.activeElement as HTMLElement).blur();
+    }
   });
 
   describe('tabs', () => {
     it('uses the IDs passed in for the tabs', () => {
-      const wrapper = mountWithAppProvider(<Tabs selected={0} tabs={tabs} />);
+      const wrapper = mountWithAppProvider(<Tabs {...mockProps} />);
 
       tabs.forEach((tab, index) => {
```

Comment:
I would maybe just pass in an ID for one tab and check the first Tab so you can avoid this loop.